helm_deploy/values-dev.yaml

```diff
@@ -18,6 +18,7 @@ generic-service:
     BOOKING_NOTICE_PERIOD_MAXIMUM_DAYS: 50
     TASK_VISIT_COUNTS_REPORT_CRON: "0 0 0/1 * * ?" #every hour on dev
     TASK_FLAG_VISITS_CRON: "0 0 0/4 * * ?" #every 4 hours on dev
+    FEATURE_REQUEST-BOOKING-ENABLED: false
```

Comment:
Is it worth enabling this on dev/staging? I've got a similar flag on the front-end to control whether to show booked/requested confirmation given `visitSubStatus` - could test end-to-end if enabled in both places?

Appreciate that until we have request review they won't be processed, but does that matter on dev/staging?